docs/guidelines/templates/templates-type.md

```diff
@@ -12,51 +12,53 @@ This table contains templates for general documents, such as `NOTES.txt` and `RE
 
 | Repository | Template | Usage | Author |
 |---|---|---|---|
-| [`kyma`](https://github.com/kyma-project/kyma)| [`NOTES.txt`](https://github.com/kyma-project/community/blob/main/guidelines/templates/resources/NOTES.txt) | Use the template for charts. | All contributors |
-| [`kyma`](https://github.com/kyma-project/kyma)| [`README.md`](https://github.com/kyma-project/community/blob/main/guidelines/templates/resources/chart_README.md) | Use the template only for internal charts with in-house components. | All contributors |
-| [`examples`](https://github.com/kyma-project/examples) | [`README.md`](https://github.com/kyma-project/community/blob/main/guidelines/templates/resources/example_README.md) | Use the template for examples. | All contributors |
+| [`kyma`](https://github.com/kyma-project/kyma)| [`NOTES.txt`](https://github.com/kyma-project/community/blob/main/templates/resources/NOTES.txt) | Use the template for charts. | All contributors |
+| [`kyma`](https://github.com/kyma-project/kyma)| [`README.md`](https://github.com/kyma-project/community/blob/main/templates/resources/chart_README.md) | Use the template only for internal charts with in-house components. | All contributors |
+| [`examples`](https://github.com/kyma-project/examples) | [`README.md`](https://github.com/kyma-project/community/blob/main/templates/resources/example_README.md) | Use the template for examples. | All contributors |
 | All | [`README.md`](https://github.com/kyma-project/community/blob/main/guidelines/repository-guidelines/repository-template/README.md) | Use the template for the `README.md` document in any repository, directory, or subdirectory within the Kyma organization. | All contributors |
```

Comment:
Please update the link - Maja found it during the last SIG Content :) We also agreed that only one general README.md template would be sufficient.